layouts/shortcodes/thirdparty-content.html

```diff
@@ -0,0 +1,3 @@
+{{ with .Parent }}
+  {{ T "thirdparty_message" | safeHTML }}
+{{ end }}
```

Comment:
Here's how I'd do it:
```
<blockquote class="3rdparty">
  <div><strong>{{ T "thirdparty_callout" }}</strong> {{ T "thirdparty_message" | markdownify }}</div>
</blockquote>
```

(and then if taking that route I would revise PR #23029).

This is easier on localization teams, I think, because it falls back to the English language message if omitted, and it doesn't trigger work to revise many pages in localizations.

(compare with switching to Hugo alerts instead of our existing callout shortcodes; if we did that switch we'd need to update every callout in every localization).

Reply:
I am not suggesting that the site use the "alert" shortcode for every admonition/callout -- just the third party content info label. This shortcode permits a configurable title.

Reply:
I like the idea of consistent styling though: one look for all callouts.

Reply:
OK. The third party content shortcode is comparable to a static label. The site's note and warning shortcodes accept text as the body of these shortcodes.
The theme's alert shortcode looks slightly different than the site's note and warning shortcodes, but provides a similar interface (you can add text, a title, set the color of the bar).
In this case, I think what you suggest @sftim makes sense.
The text could be Markdown or HTML. I see that a number of the strings in the i18n toml files contain raw HTML.
Not clear whether a convention has been established.
Here is an example (created from the alert code and some sample strings) that I tested locally.

```
{{ $color := .Get "color" | default "primary" }}
<div class="alert alert-{{ $color }}" role="alert">
<h4 class="alert-heading">{{ T "caution" | safeHTML }}</h4>
{{ T "main_cncf_project" | markdownify }}
</div>
```
To use on page:
```
{{% thirdparty-content %}}
```

Reply:
Latest commit implements @kbhawkey's review (side note: if you give me your email, I'll add you as a co-author :)) 

That said: the styling is somewhat broken at this point – it might be best to merge @sftim's https://github.com/kubernetes/website/pull/23029 and then rebase this so it can receive the correct styling via the `callout` class that PR implements. I'm PR Wrangling this week, so.... :) _skips over_

Reply:
I prefer the theme's styling of the callouts (no border, less heavy left hand border).
I understand that every callout should be one of the three categories,
but in this case, I think a new title adds clarity. I can't tell whether I am being cautioned against reading
the content or adding/updating the content.
I could see reusing this type of "labeller" shortcode for other things such as the feature state shortcode.
I have not read through pull request 23029, but would consider trying to leverage styling and templates offered
by the theme.